### ISSUE TITLE
Inline code is getting truncated

### ISSUE THREAD

**Issue by ocrickard** [lise-henry/crowbook#42]
Fantastic project! I'm working on converting a book that we had previously written in InDesign back to markdown, and it looks like the default PDF formatting is resulting in incorrect layout:

<img width="775" alt="screen shot 2017-09-29 at 9 01 25 am" src="https://user-images.githubusercontent.com/857665/31017028-22baa9fa-a4f5-11e7-845a-433ac3c171c9.png">

Here's the example paragraph in the screenshot if you want to test this:
```
A good way to think about the difference between this method and `didUpdateComponent` is that `didMount` is an opportunity for configuration of the **view**, while `didUpdateComponent` and `initWithComponent:` is about configuring the **controller itself**. The one major exception is that `initWithComponent:` is called on a background thread, so it’s common to wait for the first mounting to complete configuration that had to be done on the main thread. This is something we’d like to fix.
```

**Comment by ocrickard** [comment]
Any chance of a new release on cargo?

**Comment by lise-henry** [comment]
I'm currently changing quite a few things in the master branch, but once it is done and it has been tested a bit I will probably release 0.14.0 soon-ishly :)

**Comment by lise-henry** [comment]
Actually, I just published 0.14.0-beta to crates.io. Obviously not very tested/polished though.

**Comment by ocrickard** [comment]
Great, thank you!
